openfl/transport/grpc/aggregator_client.py

```diff
@@ -454,3 +454,42 @@ def admin_set_straggler_cutoff_time(self, admin_name, timeout_in_seconds):
         )
         response = self.stub.SetStragglerCuttoffTime(request)
         self.validate_response(response, admin_name)
+        
+    @_handle_grpc_error
+    @_atomic_connection  # MS-TODO: remove this wrapper?
+    def admin_get_(self, admin_name, timeout_in_seconds):
```

Comment:
This looks like a copy/paste failure on my part. This should be removed.

Reply:
Done